origin-dapp/src/actions/Listing.js

```diff
@@ -8,11 +8,19 @@ export const ListingConstants = keyMirror(
   {
     FETCH_IDS: null,
     FETCH_IDS_SUCCESS: null,
-    FETCH_IDS_ERROR: null
+    FETCH_IDS_ERROR: null,
+    IS_MULTI_UNIT_LISTING: null
```

Comment:
@sparrowDom what do you think about storing this in the listing-create component state? Or better yet, can this be computed based on the the step 2 form state, or is it affected by more than just the quantity field?

Reply:
Yup good thinking. I remember hitting a wall, not knowing how to pass this information down to `price-field` since it doesn't get the full scope of `formData` to access the quantity value. But then I see there is a `formContext` field in the react form, that passes the object to all the children fields like `price-field`. Using that and avoiding storing values in Redux.